scripts/printPkgPrNewPublishDirs.mts

```diff
@@ -0,0 +1,19 @@
+const publishDirs = {
+  '@mui/x-license': './packages/x-license/build',
+  '@mui/x-telemetry': './packages/x-telemetry/build',
+  '@mui/x-data-grid': './packages/x-data-grid/build',
+  '@mui/x-data-grid-pro': './packages/x-data-grid-pro/build',
+  '@mui/x-data-grid-premium': './packages/x-data-grid-premium/build',
+  '@mui/x-data-grid-generator': './packages/x-data-grid-generator/build',
+  '@mui/x-date-pickers': './packages/x-date-pickers/build',
+  '@mui/x-date-pickers-pro': './packages/x-date-pickers-pro/build',
+  '@mui/x-charts': './packages/x-charts/build',
+  '@mui/x-charts-vendor': './packages/x-charts-vendor',
+  '@mui/x-charts-pro': './packages/x-charts-pro/build',
+  '@mui/x-tree-view': './packages/x-tree-view/build',
+  '@mui/x-tree-view-pro': './packages/x-tree-view-pro/build',
+  '@mui/x-internals': './packages/x-internals/build',
+};
+
+// eslint-disable-next-line no-console
+console.log(Object.values(publishDirs).join(' '));
```

Comment:
```suggestion

```suggestion
// console.log is supposedly adding a line break, which might not be something we want here.
process.stdout.write(Object.values(publishDirs).join(' '));
```
```